src/components/event_preview.js

```diff
@@ -148,7 +173,7 @@ export default class EventPreview extends React.Component {
     if (this.props.event.event_free_text) {
       eventOptionsRows.splice(0, 0, (
         <tr>
-          <th>Free Text</th>
+          <th>Free Text asdfasdf</th>
```

Comment:
What is the purpose of this?

Reply:
Removing, sorry! I added this while I was searching for the event list code on the replay screen.